global.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Remove this file. We want to use the latest version of the SDK tools, so no global.json file is needed